### PR TITLE
Hyperledger AnonCreds Project Report 2023Q4

### PR DESCRIPTION
**Pull request by swcurran**
FYI: @mikelodder7 @aritroCoder @andrewwhitehead @berendsliedrecht @genaris @Patrik-Stas

### CODE REVIEW

**Review by petermetz — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-AnonCreds.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+objects and operations will remain in AnonCreds v2, the v1 specification is a
+huge head start towards the creation the v2 specification. Huge thanks and
+appreciation of the efforts by Aritra who has done an outstanding job!
+
+[Hyperledger Mentorship program]: https://wiki.hyperledger.org/display/INTERN/Hyperledger+Mentorship+Program
+[Aritra Bhaduri]: https://github.com/aritroCoder
+[AnonCreds Mentorship]: https://wiki.hyperledger.org/display/INTERN/Document%2C+Review%2C+and+Implement+Hyperledger+AnonCreds+ZKP+Cryptographic+Primitives
+
+The contribution of the new AnonCreds v2 code base, and underlying cryptography
+in [Hyperledger Labs Agora] represents a big opportunity to advance AnonCreds
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q4-Hyperledger-AnonCreds.md]:
```suggestion
in [Agora], a Hyperledger Lab represents a big opportunity to advance AnonCreds
```

File: project-reports/2023/2023-Q4-Hyperledger-AnonCreds.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+and the idea of maximum privacy-preserving verifiable credentials for everyday
+use. We need to assemble the set of collaborators looking to accomplish that
+goal that are willing to have developers do the work needed. A presentation at
+[Internet Identity Workshop] got good interest, going over the capabilities
+enabled by AnonCreds v2, and a "[To Do List]" of things we as a community need to
+work on.
+
+[Hyperledger Labs Agora]: https://labs.hyperledger.org/labs/agora.html
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q4-Hyperledger-AnonCreds.md]:
```suggestion
[Agora]: https://labs.hyperledger.org/labs/agora.html
```

File: project-reports/2023/2023-Q4-Hyperledger-AnonCreds.md (hunk 1 above)
Comment (tkuhrt) [reply project-reports/2023/2023-Q4-Hyperledger-AnonCreds.md]: In line with "Labs Representation" of the [Lab Proposal](https://docs.google.com/document/d/1VbgmlIqDnUnvjxSJ5QTUBV-Qn3eA7Gduw0edjYgDXIY/edit)

**Review by tkuhrt — CHANGES_REQUESTED** [review]
Sorry for being pedantic.

File: project-reports/2023/2023-Q4-Hyperledger-AnonCreds.md (hunk 2 above)
Comment (swcurran) [reply project-reports/2023/2023-Q4-Hyperledger-AnonCreds.md]: Updated

File: project-reports/2023/2023-Q4-Hyperledger-AnonCreds.md (hunk 1 above)
Comment (swcurran) [reply project-reports/2023/2023-Q4-Hyperledger-AnonCreds.md]: Updated.

**Review by tkuhrt — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by lehors — APPROVED** [review]

**Review by bobbi28 — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Pull request merged**